File: 백준/Bronze/9243. 파일 완전 삭제/파일 완전 삭제.py
N = int(input())
S = input()
E = input()
D = ''
for s in S:
    if N%2:
        if s == '1':
            D += '0'
        else:
            D += '1'
    else:
        D += s
if D != E:
    print('Deletion failed')
else:
    print('Deletion succeeded')

File: 백준/Bronze/9243. 파일 완전 삭제/파일 완전 삭제.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('', (inputN) => {
  const N = parseInt(inputN);
  
  rl.question('', (S) => {
    rl.question('', (E) => {
      let D = '';
      for (let i = 0; i < S.length; i++) {
        if (N % 2) {
          D += (S[i] === '1') ? '0' : '1';
        } else {
          D += S[i];
        }
      }

      if (D !== E) {
        console.log('Deletion failed');
      } else {
        console.log('Deletion succeeded');
      }

      rl.close();
    });
  });
});
